packages/svelte-vscode/src/html/autoClose.ts

```diff
@@ -67,17 +60,23 @@ export function activateTagClosing(
         }
         const lastChange = changes[changes.length - 1];
         const lastCharacter = lastChange.text[lastChange.text.length - 1];
-        if ('range' in lastChange && (lastChange.rangeLength ?? 0) > 0 || (lastCharacter !== '>' && lastCharacter !== '/')) {
+        if (
+            ('range' in lastChange && (lastChange.rangeLength ?? 0) > 0) ||
+            (lastCharacter !== '>' && lastCharacter !== '/')
```

Comment:
I've ran the tests locally and double checked everything, this is the only thing I'm uncertain since it adds another parentheses for the first two conditionals. It should be the same, except when `'range' in lastChange` is false, it will still check the next conditional after the `||`, while previously it would fail immediately?

Reply:
`&&` has priority over `||` so this should be the same